guide/case-studies/saving-satoshi.md

```diff
@@ -0,0 +1,433 @@
+
+While there weren’t any major changes to the plot, this final revision resulted in many smaller updates and one complete character redesign. User testing revealed that players wanted a more interesting character so after many rejected name proposals like Jessica Soup, Mike Ramen was replaced with the confident and adventurous Mika 3000.
+
+{% include picture.html
+   image = "assets/images/guide/case-studies/saving-satoshi/jessica-soup.jpg"
+   retina = "assets/images/guide/case-studies/saving-satoshi/jessica-soup@2x.jpg"
+   alt-text = "Screenshot of a chat message from Christoph at 9/17/24, 8:37 AM asking for opinions on the name 'Jessica Soup' and calling it goofy."
+   width = 800
+   height = 194
+   layout = "full-width"
+%}
+
+A number of plot holes were also addressed at this time, some that had been there since the very beginning. These were cans that could no longer be kicked down the road, calling for deep collaboration between writers and bitcoin tech experts. Care had to be put into making sure the scenarios we were creating were not only technically sound, but that they still meaningfully drove the story forward. Many things were hashed out over phone calls and long comment threads.
+
+**World building**  
+An especially fun part of story writing is creating a universe unique to the game. The team had fun reciting Holocat’s sassy one liners, imagining what sitting at a ButtLift Hover Desk would be like, and agreeing that no one would ever think to look for Satoshi’s keys on a CD-ROM labeled *Creed – My Own Prison*. 
+
+The universe really came to life at the first Saving Satoshi workshop where swag was created to represent different elements of the game like logos for the mining groups featured in chapter 3 and holographic Holocat stickers.
+
+{% include picture.html
+   image = "assets/images/guide/case-studies/saving-satoshi/swag.jpg"
+   retina = "assets/images/guide/case-studies/saving-satoshi/swag@2x.jpg"
+   mobile = "assets/images/guide/case-studies/saving-satoshi/swag-mobile.jpg"
+   mobileRetina = "assets/images/guide/case-studies/saving-satoshi/swag-mobile@2x.jpg"
+   alt-text = "Collection of Saving Satoshi merchandise including holographic cat stickers, 'I saved Satoshi' stickers, 'Thomas Vanderpoole is not Satoshi' labels, and Creed CD-ROM stickers."
+   caption = "Swag created for the TABConf 2024 workshop."
+   width = 800
+   height = 503
+   layout = "full-width"
+%}
+
+While there were a number of things that didn’t make it into the final cut of the story these ideas have been saved for later releases. Look forward to a future update with a list of bitcoin themed cocktail drinks and much more!
+
+**Incorporating bitcoin culture and history**
```

Comment:
I'm not sure why but there is no line break after this in the preview

Reply:
Good catch. I fixed those issues (with extra line breaks). Main problem is that this is bold text, when it should be a header. We have several other instances like this. I'll take a look tomorrow and update if necessary.

Reply:
Ah, got it! I have no opinion on the formatting for this (bold or header) so feel free to change as you see fit!

Reply:
Headers are the better semantic HTML choice, so I switched to those. Should all look as expected now.